Winium/Winium.Mobile.Driver/Automator/Capabilities.cs

```diff
@@ -37,6 +37,7 @@ internal Capabilities()
             this.PingTimeout = DefaultPingTimeout;
             this.NoFallback = true;
             this.CommandSettings = new CommandSettings();
+            this.IsJavascriptEnabled = true;
```

Comment:
Is it required to call ExecuteScript in some selenium bindings? In python selenium bindings do not check this flag at all and allow calling ExecuteScript commands anyway.

It is minor, but will be nice to check this, because it implies javascript support, not some custom "scripting" support.